hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
Please do not include unrelated change.  You didn't branch from `hw1`.

Since it's close to deadline, I suggest you to force push (and optionally squash) so that you do not need to use a separate PR.